src/nodejs/node_osrm.cpp

```diff
@@ -167,13 +167,13 @@ inline void async(const Nan::FunctionCallbackInfo<v8::Value> &info,
                 ParseResult(status, json_result);
                 if (pluginParams.renderToBuffer)
                 {
-                    std::ostringstream buf;
-                    osrm::util::json::render(buf, json_result);
-                    result = buf.str();
+                    std::string json_string;
```

Comment:
Started using `std::string` here as it is faster. `osrm-routed` uses `std::vector`.

Reply:
We probably can even try to go with `std::vector` here too(which is slightly faster according to benchmarks).